src/main/java/nl/hsac/fitnesse/symbols/RandomString.java

```diff
@@ -41,13 +41,56 @@ public RandomString() {
     }
 
     public String toTarget(Translator translator, Symbol symbol) {
+        String param = symbol.getProperty(MIN_MAX_LENGTH, null);
```

Comment:
Why not use the same default value as for prefix (i.e. empty string `""`, or change that default to `null` also)